lib/process-services-cloud/src/lib/people/components/people-cloud.component.ts

```diff
@@ -123,6 +123,9 @@ export class PeopleCloudComponent implements OnInit, OnChanges, OnDestroy {
     @Input()
     title: string;
 
+    @Input()
+    minNrOfCharacters?: number;
```

Comment:
Why do we want this check configurable? i mean we want to avoid making the calls for empty string that is something normal, is there a scenario where this makes sense?

Reply:
This component is used in many places and I was not sure if it is not going to break anything. So this is why I made this optional.

Reply:
removed optional property